public/js/santaAdmin.js

```diff
@@ -81,14 +82,15 @@ $(async function() {
       $('#forbiddenUser1, #forbiddenUser2').append(`<option value="${friend._id}" data-email="${friend.email}">${friend.name}</option>`);
     });
   });
-
   $('#forbiddenPairsForm').on('submit', () => {
     const pair = {
       forbiddenUser1Id: $('#forbiddenUser1').val(),
       forbiddenUser2Id: $('#forbiddenUser2').val()
     };
     $.post('api/forbidden', pair, result => {
-      showAlert(result);
```

Comment:
Why did you remove this?

Reply:
Probably trying to get the code right, I don't have a specific reason. This is a mistake.